tic_tac_toe/player.py

```diff
@@ -0,0 +1,12 @@
+class Player:
+    def __init__(self, token):
+        self.token = token
+
+
+class Human(Player):
```

Comment:
What does inheritance get you here?

Reply:
@mikeknep This was in preparation for implementing a Computer player. I was thinking of the Human and Computer players as two instances of the same thing, so it made sense to me that they would inherit from a Player superclass.

On the other hand, I also thought about the possibility that it was kind of overly complicated given the fact that the only shared trait players have is the token.